src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -97,70 +100,72 @@
            can-edit? (common/access-greater-than? (:access-level props) "READER")
            config-id (ws-common/config->id config)]
-       [:div {:style {:width 290 :float "left"}}
-        [:div {:ref "sidebar"}]
-        (apply style/create-unselectable :div
-               {:style {:position (when-not (:sidebar-visible? @state) "fixed")
-                        :top (when-not (:sidebar-visible? @state) 4)
-                        :width 290}}
-               (if editing?
-                 (list
-                  [comps/SidebarButton {:color :success-state
-                                        :text "Save" :icon :done
-                                        :onClick #(this :-commit)}]
-                  [comps/SidebarButton {:color :exception-state :margin :top
-                                        :text "Cancel Editing" :icon :cancel
-                                        :onClick #(this :-cancel-editing)}])
-                 (conj
-                  (when can-edit?
-                    [[comps/SidebarButton {:style :light :color :button-primary
-                                           :text "Edit Configuration" :icon :edit
-                                           :disabled? (when locked? "The workspace is locked")
-                                           :onClick #(this :-begin-editing)}]
-                     [comps/SidebarButton {:style :light :color :exception-state :margin :top
-                                           :text "Delete" :icon :delete
-                                           :disabled? (when locked? "The workspace is locked")
-                                           :onClick #(modal/push-modal
-                                                      [delete/DeleteDialog {:config-id config-id
-                                                                            :workspace-id (:workspace-id props)
-                                                                            :after-delete (:after-delete props)}])}]])
-                  [comps/SidebarButton {:style :light :color :button-primary :margin (when can-edit? :top)
-                                        :text "Publish..." :icon :share
+       [:div {:style {:flex "0 0 270px" :paddingRight 30}}
+        [Sticky
+         {:outer-style {:width 290 :backgroundColor "#fff"}
+          :anchor body-id
+          :sticky-props {:data-check-every 1}
+          :contents
+          [:div {:style {:width 270 :background "#fff"}}
+           (if editing?
+             (list
+              [comps/SidebarButton {:color :success-state
+                                    :text "Save" :icon :done
+                                    :onClick #(this :-commit)}]
+              [comps/SidebarButton {:color :exception-state :margin :top
+                                    :text "Cancel Editing" :icon :cancel
+                                    :onClick #(this :-cancel-editing)}])
+             (list*
+              (conj
```

Comment:
Looks like we can simply `(concat ...)` rather than `(list* (conj ...))`. Concat handles nil values by ignoring them.